src/screens/EventScreens/EventDetailScreen/index.js

```diff
@@ -109,6 +130,18 @@ const EventDetailScreen = ({ navigation, route }) => {
       { text: 'Yes', onPress: confirmQuitEvent },
     ]
   );
+  const handleSave = React.useCallback(async () =>{
+    const eventInfoObj = {
```

Comment:
can simplify this object to
```
const eventInfoObj = {
      eventDescription,
      eventLocation,
      eventName,
      eventEndTime: dayjs(eventEndTime).valueOf(),
      eventStartTime: dayjs(eventStartTime).valueOf(),
      hostId: userInfo['userId'],
      id: route.params.eventId,
    };
```
see https://alligator.io/js/object-property-shorthand-es6/